src/js/models/Sign.js

```diff
@@ -1,99 +1,114 @@
-const property = value => ({
-  writable: false,
-  configurable: false,
-  enumerable: true,
-  value
-})
+import { Record } from 'immutable'
+import Model from './Model.js'
```

Comment:
I don't see Model being used in the code that follows.  Should it be deleted?

Reply:
Good call.